### PR TITLE
UM01 - Health Factors

### PR DESCRIPTION
**Pull request by Hyodar**
# Health Factors Monitoring Bot

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] New Bot
- [ ] Fix a bot error
- [ ] Update a bot logic/test/documentation
- [ ] Code style update (formatting, renaming)
- [ ] Other (please describe):

## New Bot PR readiness

- [x] `npm install` command works (i.e. package-lock is updated properly)
- [x] The `name` and `description` fields of `package.json` describe the bot properly
- [x] All the libraries listed in `package.json` are being used
- [x] All the development libraries are in `devDependencies` section of `package.json`
- [x] The bot README is updated
- [x] The bot README sufficiently describes what the bot does
- [x] All the alerts that the bot emits are listed in the README
- [x] The txn/block examples listed in the README generate the expected findings
- [x] Code is easy to follow and/or properly commented
- [x] Bot contains tests
- [x] All tests pass when running `npm test`
- [x] Tests are not making real network calls (i.e. network interaction is properly mocked)
- [x] Tests cover all the paths (i.e. negative & positive test cases)
- [x] When the tests complete, no warning/errors are shown
- [x] `npm start` works without errors when executed using a jsonRpcUrl for each of the networks specified in the README
- [x] The networks specified in the README and the networks are listed in the `package.json` are the same
- [x] The code does what the README describes
- [x] Network calls are properly being cached (if duplicated calls can occur)
- [x] A performance review has been done (e.g. `Promise.all` is used whenever possible)
- [x] The findings fields are filled out correctly
- [x] Findings `protocol` is specified (if the bot is for a specific one)
- [x] `metadata` of the findings contains all the information needed.
- [x] The SDK methods are being used appropriately (e.g. `filterLog`, `filterFunction`, `getEthersProvider`)
- [x] `ABI`s match the contracts used / `args` used from `LogDescriptions` exists in the `ABI`s
- [x] The bot follows Forta recommended [best practices](https://docs.forta.network/en/latest/best-practices/)
- [x] Bot can recover from failed error calls when they can fail (e.g. errors are being caught on calls that can fail)
- [x] `repository` is added in the `package.json`

### CODE REVIEW

**Review by 0xtaf — COMMENTED** [review]
1 high severity vulnerability at a package - `npm audit fix` is needed

File: Umee-Bots/Health-Factors/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Ethereum
+
+## Alerts
+
+- UMEE-1
+  - Fired when a large borrowing position is close to liquidation
+  - Severity is always set to "medium"
+  - Type is always set to "info"
+  - Metadata:
+    - `address`: lender address
```
Comment (0xtaf) [inline Umee-Bots/Health-Factors/README.md:19]: should it be borrower address?

**Review by yehia67 — APPROVED** [review]

File: Umee-Bots/Health-Factors/README.md (hunk 1 above)
Comment (Hyodar) [reply Umee-Bots/Health-Factors/README.md]: You're right, good catch!

**Comment by Hyodar** [comment]
Re-requesting review since the issues related to the comments were fixed.

**Review by 0xtaf — APPROVED** [review]
great work! it really teaches a lot too. 🚀

**Pull request merged**